trendmicrovisionone_connector.py

```diff
@@ -835,850 +733,1438 @@ def _handle_remove_from_blocklist(self, param):
+        Note: You can specify either the computer name ("endpointName") or the GUID of the installed agent program ("agentGuid").
+        Args:
+            process_identifiers(List[Dict[str, str]]): Object containing mailBox/messageId and optional description
+            or uniqueId and optional description.
+        Returns:
+            multi_resp(Dict[str, List]): Object containing task_id and HTTP status code.
+        """
+        self.save_progress(
+            "In action handler for: {0}".format(self.get_action_identifier())
+        )
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Required Params
+        process_identifiers: List[Dict[str, str]] = json.loads(
+            param["process_identifiers"]
+        )
+
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        multi_resp: List[MsData] = []
+
+        # Make rest call
+        for i in process_identifiers:
+            response = client.terminate_process(
+                pytmv1.ProcessTask(
+                    endpoint_name=i["endpoint"],
+                    file_sha1=i["file_sha1"],
+                    description=i.get("description", "Terminate Process."),
+                    file_name=i.get("filename", ""),
+                )
+            )
+            if "error" in response.result_code.lower():
+                self.debug_print(
+                    "Something went wrong, please check process identifiers."
+                )
+                raise RuntimeError(
+                    f"Error while terminating process: {response.errors}"
+                )
+            assert response.response is not None
+            multi_resp.append(response.response.items[0])
+
+        # Add the response into the data section
+        action_result.add_data({"multi_response": [item.dict() for item in multi_resp]})
+
+        # Return success
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def exception_list_count(self) -> int:
+        """Gets the count of objects present in exception list"""
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        new_exceptions: List[Dict[str, Any]] = []
+        try:
+            client.consume_exception_list(
+                lambda exception: new_exceptions.append(exception.dict())
+            )
+        except Exception as e:
+            self.debug_print("Consume Exception List failed with following exception:")
+            raise RuntimeError("Error while fetching exception list count.") from e
+
+        # Load json objects to list
+        exception_objects: List[Dict[str, Any]] = []
+        for i in new_exceptions:
+            i["description"] = "" if not i["description"] else i["description"]
+            i = json.dumps(i)
+            exception_objects.append(json.loads(i))
+        exception_count = len(exception_objects)
```

Comment:
Instead of doing this, there is no reason that you are actually storing the exception objects other than to get the count. However, your new_exceptions variable already tells you that information. You can avoid this by returning len(new_exceptions) given that you are not doing anything with the actual list.

Reply:
updated and working